### PR TITLE
Add response schemas and a good_mock_server for v1.3.0

### PR DESCRIPTION
**Pull request by yash-puligundla**
- Report runner does not terminate if service-info endpoint validation fails. Instead, it only terminates without finishing all the tests only when the DRS server version is not obtained from the service-info endpoint or if the obtained DRS version is not supported by the DRS compliance suite
- Add response schemas for DRS v1.3.0 in `compliance_suite/schemas/v1.3.0` directory
- Rename existing good_mock_server.py to `good_mock_server_v1.2.0.py`
- Create a new good_mock_server that follows DRS v1.3.0 - `good_mock_server_v1.3.0.py`
- Update README

### CODE REVIEW

File: compliance_suite/report_runner.py [hunk 1]
```diff
@@ -106,14 +93,22 @@ def report_runner(server_base_url, platform_name, platform_description, auth_typ
         response = response)
 
     # Get the DRS version number from service-info
-    # If the version is not supported by the compliance suite,
+    # If the version is not avaible or is supported by the compliance suite,
```
Comment (Chen2x) [inline compliance_suite/report_runner.py:96]: I think you mean 

```suggestion
    # If the version is not available or is not supported by the compliance suite,
```

**Comment by Chen2x** [comment]
I tested 1.2.0 and 1.3.0 and everything worked good. I also saw you fixed some of my leftover mistakes in 1.2.0 haha thanks

**Comment by yash-puligundla** [comment]
@Chen2x Can you please approve the PR if it looks good?

File: compliance_suite/report_runner.py (hunk 1 above)
Comment (yash-puligundla) [reply compliance_suite/report_runner.py:96]: that's right! Thanks for catching this.

**Review by Chen2x — APPROVED** [review]

**Pull request merged**